src/Semantics.js

```diff
@@ -43,6 +43,11 @@ Wrapper.prototype.toString = function() {
   return '[semantics wrapper for ' + this._node.grammar.name + ']';
 };
 
+// This is used by ohm editor to display a node wrapper appropriately.
+Wrapper.prototype.toJSON = function() {
+  return '[semantics wrapper for ' + this._node.grammar.name + ']';
```

Comment:
I talked to @pdubroy and clarified some of his concerns with the `toJSON`. This seems fine but can you just call the `toString` method here?